### PR TITLE
fix: librarian status updates

### PR DESCRIPTION
**Pull request by juleskuehn**
Fixes a bug where document processing status icons (in "documents" list for a particular data source in librarian modal) did not finish updating. i.e. "poll_url" was None in the context from the `poll_status` view even though there were ongoing tasks.

To test:
- start celery with a small number of concurrent tasks, i.e. 2
- upload some documents to a data source (more than 2)
- ensure that the status icons for all of them continues to update until all documents are complete, without any intervention

### CODE REVIEW

File: django/librarian/views.py [hunk 1]
```diff
@@ -299,7 +299,7 @@ def poll_status(request, data_source_id, document_id=None):
     documents = Document.objects.filter(data_source_id=data_source_id)
     poll = False
     try:
-        poll = documents.filter(status__in=[IN_PROGRESS_STATUSES]).exists()
+        poll = documents.filter(status__in=IN_PROGRESS_STATUSES).exists()
```
Comment (juleskuehn) [inline django/librarian/views.py:302]: This was the actual issue.

Comment (Rifahaziz) [reply django/librarian/views.py:302]: this works great! I don't have to refresh anymore to see the updated statuses :)

File: django/tests/librarian/test_librarian.py [hunk 2]
```diff
@@ -413,3 +413,79 @@ def test_modal_views(client, all_apps_user):
     # Check the library object is deleted
     assert not Library.objects.filter(id=tmp_library.id).exists()
     assert not LibraryUserRole.objects.filter(id=role.id).exists()
+
+
```
Comment (Rifahaziz) [inline django/tests/librarian/test_librarian.py:417]: new test passed and looks good!

**Review by Rifahaziz — APPROVED** [review]
All good!

**Pull request merged**